conf.py

```diff
@@ -98,6 +99,24 @@
 #
 html_theme = 'sphinx_rtd_theme'
 
+html_title = 'EPICS Documentation'
+html_short_title = 'EPICS Docs'
+html_baseurl = 'https://docs.epics-controls.org/'
+
+html_show_sphinx = False
+html_show_copyright = True
+html_copy_source = False
+html_permalinks = True
+html_permalinks_icon = '#'
```

Comment:
I think you can remove the settings which are already the defaults.

```suggestion
html_show_sphinx = False
html_copy_source = False
html_permalinks_icon = '#'
```

On a side note, is there a reason to remove the "Built with Sphinx" message? I think it's nice to have, to show appreciation to the tools you use.

Reply:
I just assumed it had been missed. It is fully transparent if inspecting the source code. Should I add it back?

Reply:
I think we should leave the "Built with Sphinx …" message, yes